app/scripts/components/common/layout-root/index.tsx

```diff
@@ -43,43 +49,16 @@ const PageBody = styled.div`
 
 function LayoutRoot(props: { children?: ReactNode }) {
   const cookieConsentContent = getCookieConsentFromVedaConfig();
-  const readCookie = (name) => {
-    const nameEQ = name + '=';
-    const attribute = document.cookie.split(';');
-    for (let i = 0; i < attribute.length; i++) {
-      let c = attribute[i];
-      while (c.charAt(0) == ' ') c = c.substring(1, c.length);
-      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length, c.length);
-    }
-    return null;
-  };
-
-  const getCookie = () => {
-    const cookie = readCookie(COOKIE_CONSENT_KEY);
-    if (cookie) {
-      const cookieContents = JSON.parse(cookie);
-      if (cookieContents.answer) setGoogleTagManager();
-      return cookieContents;
-    }
-    return NO_COOKIE;
-  };
-
-  const showForm = () => {
-    const cookieContents = getCookie();
-    if (cookieContents === NO_COOKIE) {
-      return true;
-    } else {
-      return !cookieContents.responded;
-    }
-  };
+  const bannerContent = getBannerFromVedaConfig();
   const { children } = props;
-
-
+  const [sessionStart, setSesstionStart] = useState<string | undefined>();
+  const sessionItem = window.sessionStorage.getItem(SESSION_KEY);
```

Comment:
`window` is a client side object, so this will probably fail in our Next.js instance. Let's check if `window` is defined before trying to access it.

Reply:
Was this concern addressed?